topics.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+apt update
```

Comment:
Using `apt` is too platform-specific: this script will only run on Debian-based systems out of the box.

It is better to specify `git`, `python3` and `pip` as requirements to run the visualizer and let the user take care of that themselves.

You may want to check in the script whether the binaries are present, so that the user sees an actionable message ("Please install `git`/...") rather than an error.